src/app/components/elements/PageMetadata.tsx

```diff
@@ -100,6 +103,7 @@ export const PageMetadata = (props: PageMetadataProps) => {
                         {(subtitle || doc?.subtitle) && <h5><Markup encoding="latex">{subtitle ?? doc?.subtitle}</Markup></h5>}
                     </div>}
                     {isAda && <div className="d-flex align-items-center">
+                        {isCrossTopic && <CrossTopicQuestionIndicator className="me-3"/>}
                         {pageContainsLLMFreeTextQuestion && <LLMFreeTextQuestionIndicator className="me-3"/>}
                         <EditContentButton doc={doc} />
                     </div>}
```

Comment:
If both of these indicators exist at the same time (e.g. I've changed `post_18_stage_test` so this is the case), they push the action buttons off-screen at <=md screen sizes. We should probably stack them instead of keeping them horizontal at these sizes to adjust.

That is to say change the surrounding div to `align-items-end` for Ada, then remove the `d-flex` from this div and separate the `EditContentButton` to keep it inline:
```suggestion
                    {isAda && <> <div className="d-lg-flex align-items-center">
                        {isCrossTopic && <CrossTopicQuestionIndicator className="me-3"/>}
                        {pageContainsLLMFreeTextQuestion && <LLMFreeTextQuestionIndicator className="me-3"/>}
                    </div>
                    <EditContentButton doc={doc} /> </>}
```

Don't worry about the other instance of this in the file. It's not possible for the issue to come up in that case and I have qualms with how this entire file is laid out anyway, which I will deal with in due time.